src/lib/Carousel.svelte

```diff
@@ -22,7 +22,8 @@
 {#key carouselImage}
 	<div class="header" style="background-image: url({carouselImage.src});">
 		<div class="details-container shadow">
-			<span class="name">DDD South West</span><br />
+			
+			<h1 class="name">DDD South West Conference</h1><br />
```

Comment:
I don't think we should add "Conference" to the title as I think this should match what we call ourselves - i.e. "DDD South West"

Reply:
I'm not sure where we should put it though if we remove the tag line 😅